project/plugins.sbt

```diff
@@ -10,5 +10,5 @@ addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.11.1")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.2")
 
 libraryDependencies ++= Seq(
-  "org.apache.avro" % "avro-compiler" % "1.8.2"
+  "org.apache.avro" % "avro-compiler" % "1.11.3"
```

Comment:
it makes me a bit nervous to be compiling and testing with two different versions of avro (if I'm correct that that's what happening here). is there anything preventing us from moving to avro "1.11.3" everywhere (I would guess we're blocked on something in scio)?